src/main/scala/temple/ast/Templefile.scala

```diff
@@ -19,7 +20,24 @@ case class Templefile(
   def servicesWithPorts: Iterable[(String, ServiceBlock, Ports)] =
     services
       .zip(Iterator.from(1025, step = 2)) // 1024 is reserved for auth service
```

Comment:
Hmm - surely 1025 is `ProjectConfig.authPort + 1`, which is used below? I think all of these should be coming from the same place - bring this into a `ProjectConfig` variable too?